app/Http/Controllers/ManageStocksController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Product;
+use App\Models\Stock;
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+
+class ManageStocksController extends Controller
+{
+    public function index()
+    {
+        $products = Product::with(['productType', 'productSizes', 'stocks'])->get();
+
+        return view("admin.stocks", ['products' => $products]);
```

Comment:
Dubbele haakjes gebruik je het liefste alleen voor string interpolation
```suggestion
        return view('admin.stocks', ['products' => $products]);
```